traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -31,34 +32,28 @@ sub getTenantName {
 	return defined($tenant_id) ? $self->db->resultset('Tenant')->search( { id => $tenant_id } )->get_column('name')->single() : "n/a";
 }
 
-sub isRootTenant {
-	my $self 	= shift;
-	my $tenant_id	= shift;
-	return !defined($self->db->resultset('Tenant')->search( { id => $tenant_id } )->get_column('parent_id')->single());
-}
-
 sub index {
-	my $self 	= shift;
-	my @data = ();
-	my %idnames;
+	my $self 	= shift;	
 	my $orderby = $self->param('orderby') || "name";
 
-	my $rs_data = $self->db->resultset("Tenant")->search();
-	while ( my $row = $rs_data->next ) {
-		$idnames{ $row->id } = $row->name;
-	}
+	my $tenant_utils = UI::TenantUtils->new($self);
```

Comment:
Cool:) Will do.
I was not aware of this dir, just put it next to the "Util.pm"....